apps/storybook-e2e/src/e2e/elements/select/vl-select.stories.cy.ts

```diff
@@ -1,80 +1,9 @@
-    it('should contain an option to select', () => {
+describe('story vl-select <select[is="vl-select"]> - default', () => {
+    it('should display story', () => {
         cy.visit(selectUrl);
-
-        cy.get('[is="vl-select"]').select('Frankrijk').should('have.value', 'Frankrijk');
-    });
-
-    it('should contain select with an error state', () => {
-        cy.visit(`${selectUrl}&args=error:true`);
-
-        cy.get('[is="vl-select"]').should('have.class', 'vl-select--error');
-    });
-
-    it('should contain select with a success state', () => {
-        cy.visit(`${selectUrl}&args=success:true`);
-
-        cy.get('[is="vl-select"]').should('have.class', 'vl-select--success');
-    });
-
-    it('should contain select as a block level element', () => {
-        cy.visit(`${selectUrl}&args=block:true`);
-
-        cy.get('[is="vl-select"]').should('have.class', 'vl-select--block');
-    });
-
-    it('should contain a disabled select', () => {
-        cy.visit(`${selectUrl}&args=disabled:true`);
-
-        cy.get('[is="vl-select"]').should('have.class', 'vl-select--disabled');
-    });
-
-    it('should contain a select with search functionality', () => {
-        cy.visit(`${selectUrl}&args=select:true`);
-
-        cy.get('[is="vl-select"]')
-            .parent()
-            .parent()
-            .should('have.class', 'js-vl-select')
-            .should('have.attr', 'aria-expanded', 'false')
-            .click()
-            .should('have.class', 'is-open')
-            .should('have.attr', 'aria-expanded', 'true')
-            .find('.vl-select__item')
-            .contains('België');
-
-        cy.get('[is="vl-select"]')
-            .parent()
-            .parent()
-            .find('.vl-select__list')
-            .should('have.attr', 'aria-expanded', 'true')
-            .children()
-            .eq(1)
-            .click();
-
-        cy.get('[is="vl-select"]').parent().parent().find('.vl-select__item').contains('Duitsland');
-    });
-
-    it('should disable search for extended functionality', () => {
-        cy.visit(`${selectUrl}&args=select:true;selectDisableSearch:true`);
-
-        cy.get('[is="vl-select"]').parent().parent().find('input.vl-input-field').should('not.exist');
-    });
-
-    it('should position options for extended functionality above select element', () => {
-        cy.visit(`${selectUrl}&args=select:true;position:top`);
-
-        cy.get('[is="vl-select"]').parent().parent().click();
-        cy.get('[is="vl-select"]').parent().parent().should('have.class', 'is-flipped');
-    });
-
-    it('should position options for extended functionality below select element', () => {
-        cy.visit(`${selectUrl}&args=select:true;position:bottom`);
-
-        cy.get('[is="vl-select"]').parent().parent().click();
-        cy.get('[is="vl-select"]').parent().parent().should('not.have.class', 'is-flipped');
+        cy.get('select[is="vl-select"]').select('Frankrijk').should('have.value', 'Frankrijk');
```

Comment:
Dit gaat idd testen of de `select[is="vl-select"]` value `Frankrijk` heeft; maar dit gaat even goed werken op een native `select` dus dit geeft geen zekerheid dat de `vl-select` web component geregistreerd is.